docker-java/src/test/java/com/github/dockerjava/cmd/StartContainerCmdIT.java

```diff
@@ -543,6 +543,29 @@ public void startContainerWithRestartPolicy() throws DockerException {
         assertThat(inspectContainerResponse.getHostConfig().getRestartPolicy(), is(equalTo(restartPolicy)));
     }
 
+    @Test
+    public void startContainerWithNanoCPUs() throws DockerException {
```

Comment:
given that we have `createContainerWithNanoCPUs`, I believe this test is not needed